### PR TITLE
Add s3gw-v0.15.0.md and update submodules to v0.15.0

### PR DESCRIPTION
**Pull request by giubacc**
# Describe your changes

Add s3gw-v0.15.0.md and update submodules to v0.15.0

Signed-off-by: Giuseppe Baccini <giuseppe.baccini@suse.com>

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] CHANGELOG.md has been updated should there be relevant changes in this PR.

### CODE REVIEW

**Review by m-ildefons — APPROVED** [review]
LGTM

File: docs/release-notes/s3gw-v0.15.0.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Release Notes - v0.15.0
+
+This release focuses on stabilizing our continuous integration and release process.
+In this context, we have also addressed a number of issues that was affecting our
+testing framework when automatically triggered by CI.
+
+This activity, although not introducing a direct improvement consumable by the user,
```
Comment (jhmarina) [inline docs/release-notes/s3gw-v0.15.0.md:7]: "Although this activity may not result in any direct user-facing improvements, it plays a crucial role in maintaining a stable environment for the upcoming major enhancements that the s3gw team is currently developing."

File: docs/release-notes/s3gw-v0.15.0.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+is crucial to ensure a proper and stable environment for the upcoming major
+enhancements the s3gw's team is currently working on.
+
+Obviously, we still continued to address regular issues affecting all the s3gw's
```
Comment (jhmarina) [inline docs/release-notes/s3gw-v0.15.0.md:11]: I'd get rid of "Obviously", perhaps something like "We continue to address the regular issues that affect all of s3gw's components."

**Review by jhmarina — CHANGES_REQUESTED** [review]
A couple of changes to the text, other than that, lgtm

**Pull request merged**